modules/book.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable  */
+const AwsomeBkForm = document.querySelector('.AwsomeBkForm'); 
+const subTitle = document.querySelector('#sub--title');
```

Comment:
- [ ] Optional
- Hint: `subTtle` is declared, But its value is never read. Your Linter mistake might come from here. I would suggest you look at it carefully.